scheds/rust/scx_bpfland/src/bpf/main.bpf.c

```diff
@@ -1282,5 +1296,6 @@ SCX_OPS_DEFINE(bpfland_ops,
 	       .init_task		= (void *)bpfland_init_task,
 	       .init			= (void *)bpfland_init,
 	       .exit			= (void *)bpfland_exit,
+	       .flags			= SCX_OPS_ENQ_LAST,
```

Comment:
Does setting this flag make any behavior difference? If the expired slice is always replenished in `.dispatch()` the auto re-enqueue is actually the behavior that would match new kernel behavior (as tasks that keep running won't go through `enqueue` at all).

Reply:
Actually I'm not noticing any behavior change with / without the flag. IIUC the auto re-enqueue if triggered if I don't consume any other task in `ops.dispatch()` and the prev->scx.slice isn't replenished. But if I replenish the time slice I won't trigger the auto re-enqueue anyway, so basically I don't have to set the `SCX_OPS_ENQ_LAST` flag.

Reply:
Yeah, the only difference would be whether you get `.enqueue` call if you forget to set the slice. Given that that's not supposed to happen and the kernel will be changed to warn on that, leaving it off probably makes more sense.

Reply:
ack, flag dropped, commit updated, will do more testing, but it seems to work as expected so far.